examples/Readme

```diff
@@ -0,0 +1 @@
+# WIP
```

Comment:
Add both files to the directory noted in the JIRA

Reply:
Completely missed that comment - moving now